Makefile

```diff
@@ -78,13 +78,13 @@ specs-rococo-local:
 
 .PHONY: format
 format:
-	cargo +nightly-2023-07-13 fmt
+	cargo +nightly fmt
```

Comment:
Do we need to make this change over on the other github workflows? (codecov, release, and verify-pr-commit)

Those are all still constrained to this particular version.